lib/arena.rb

```diff
@@ -1,2 +1,49 @@
 class Arena
+
+  attr_accessor :name, :gladiators
+
+  def initialize(name)
+    @name = name.capitalize
+    @gladiators = []
+  end
+
+  def add_gladiator(gladiator)
+    @gladiators << gladiator
+    # puts "adding gladiator #{gladiator}"
+  end
+
+  def add_gladiator(gladiator)
+    if @gladiators.length < 2
+      @gladiators << gladiator
+    end
+  end
+
+  def fight # no gladiators, does nothing
+  end
+
+  def start_fight(weapon)
+    if @gladiators.length == 2
+      #compare weapons (tridentVspear, clubVspear, tridentVclub, sameVsame)
+
+      #create if statements with weapon combinations
+      if @gladiators.weapon(0) && @gladiators.weapon(1)
```

Comment:
This code is attempting to call `weapon` on the array of gladiators.  Put another way, you are asking the all gladiators for "weapon".  Imagine standing in front of a group of gladiators, putting out your hand, and ordering "weapon".  They would be confused.  However, if you stood in front of the group and pointed at the first gladiator (`gladiator = gladiators.first` or `gladiator = gladiators[0]`) and asked her for her weapon (gladiator.weapon), then you would probably get what you want.